index.html

```diff
@@ -0,0 +1,91 @@
+
+
+<!DOCTYPE html>
+<html>
+    <head>
+       <link href="./style.css" rel="stylesheet">
+        <title>Tribute</title>
+    </head>
+    <body>
+        <main id='main'>
+            <h1 id='title'>Dr. Norman Borlaug</h1>
+            <div id="img-div">
+                <p id='img-caption'>The man who saved a billion lives</p>
+                <div id="img">
+                    <p id="imge">
+                    <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+                <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p><br><br>
+                </p>
+                </div>
+                <p><h2>Here's a time line of Dr. Borlaug's life:</h2></p>
+            </div>
+            
```

Comment:
- [ ] Great job spotting 👍  this issue. But as I mentioned before, it will be great to use an explanation for each requested change. How about if you include something like: 
  - "The code looks good, I think it would be better to use semantic HTML tags. That makes the HTML more comprehensible by better defining the different sections and layout of web pages."